Dockerfile

```diff
@@ -10,7 +10,7 @@ LABEL maintainer="Jacob Alberty <jacob.alberty@foundigital.com>"
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-ARG PKGURL=https://dl.ui.com/unifi/7.5.174/unifi_sysvinit_all.deb
+ARG PKGURL=https://dl.ui.com/unifi/7.5.187-f57f5bf7ab/unifi_sysvinit_all.deb
```

Comment:
since .187 is official now, the url needs to be updated again: 

```
$ curl -I https://dl.ui.com/unifi/7.5.187/unifi_sysvinit_all.deb
HTTP/1.1 200 OK
```